src/css/override_vars.scss

```diff
@@ -23,29 +23,89 @@ $color-secondary-lightest:   #deeeef; // DEEF!
 $color-focus:                rgba(255, 221, 0, 0.55);
 $color-visited:              #4c2c92;
 
-$color-gold:                 #f8ba1c;
+$color-gold:                 #f4b400;
 
-$color-finish:               #0daf02;
+$color-finish:               #0daf00;
 $color-warning:              #cb4f03;
-$color-error:                #f16031;
+$color-error:                #cb4f03;
+$color-red:                #cb4f03;
 
 $color-gray:                 #595959;
-$color-gray-dark:            darken($color-gray, 20%);
+$color-darkgray: #555;
+$color-mediumgray: #8e8e8e;
+$color-lightgray: #d3d3d3;
+$color-textblack: #333;
+$color-trueblack: #000;
+
+$color-gray-dark: #555; // redundant to prevent breaking
 
 $color-disclaimer-text:      #000000;
 $color-disclaimer-link:      #0071bc;
 
-$color-base:                 $color-gray;
-$text-max-width:             660px;
+$color-base:                 $color-textblack;
+$text-max-width:             36em;
 
 // TODO what color is this? it should follow convention
-$color-lightgray:            #bababa;
 $site-max-width:             1040px;
 
 $color-background-light:     #f3f4f8;
+$color-cream:     #f5f5f3;
 
 $nav-width:                  600px;
 
 $font-sans:                  '18Franklin', system, -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
 $font-serif:                 'Georgia', 'Cambria', 'Times New Roman', 'Times', serif;
 $font-mono:                  'Roboto Mono', 'Bitstream Vera Sans Mono', 'Consolas', 'Courier', monospace;
+
+$base: 8;
+$base-grid: #{$base}px;
+$base-rem: #{$base}rem;
+$base-type: $base * 2;
+$base-type-rem: #{$base * 2}rem;
+$base-type-px: #{$base * 2}px;
+
+$bx5: #{($base / 2 / $base-type)}rem;    // 4px
+$b1x: #{($base / $base-type)}rem;        // 8px
+$b2x: #{($base * 2 / $base-type)}rem;   // 16px
+$b3x: #{($base * 3 / $base-type)}rem;   // 24px
+$b4x: #{($base * 4 / $base-type)}rem;   // 32px
+$b5x: #{($base * 5 / $base-type)}rem;   // 40px
+$b6x: #{($base * 6 / $base-type)}rem;   // 48px
+$b7x: #{($base * 7 / $base-type)}rem;   // 56px
+$b8x: #{($base * 8 / $base-type)}rem;   // 64px
+$b9x: #{($base * 9 / $base-type)}rem;   // 72px
+$b10x: #{($base * 10 / $base-type)}rem; // 80px
+
+$sans-s1: (8rem / $base-type);    // 8px;
```

Comment:
Actually, it may still be the opposite of basscss (whose shortcode I don't really like for type scale...) IDK, maybe it should go the other way... but I like to think bigger number equals bigger face. I'm just one man, tho!